docs/source/autograd.rst

```diff
@@ -11,6 +11,37 @@ Automatic differentiation package - torch.autograd
 
 .. autofunction:: grad
 
+.. _functional-api:
+
+Functional higher level API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+    This API is experimental.
```

Comment:
nit: I'd put a little more info on what experimental means -- that the performance still needs to be improved and the API might change (although the later is unlikely).